### PR TITLE
Kogito ui

### PR DESCRIPTION
**Pull request by AjayJagan**
Added Login Page and the landing page for Kogito UI using React and Patternfly 4.

### CODE REVIEW

File: src/components/DashboardComponent/Dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Page, SkipToContent } from '@patternfly/react-core';
+import './Dashboard.css';
+import HeaderComponent from '../PageHeaderComponent/HeaderComponent';
+// import { Route } from 'react-router-dom';
```
Comment (domhanak) [inline src/components/DashboardComponent/Dashboard.tsx:5]: Please remove the commented out code.

File: src/components/PageToolbarComponent/PageToolbarComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    const kebabDropdownItems = [
+      <DropdownItem>
+        <BellIcon /> Notifications
+      </DropdownItem>,
+      <DropdownItem>
+        <CogIcon /> Settings
+      </DropdownItem>
+    ];
+    const userDropdownItems = [
+      //<DropdownItem component={AboutModalBox}>
```
Comment (domhanak) [inline src/components/PageToolbarComponent/PageToolbarComponent.tsx:68]: same here

**Review by domhanak — CHANGES_REQUESTED** [review]

**Comment by domhanak** [comment]
Build is failing because of new firewall settings on CI. Gonna look into that.

**Comment by ederign** [comment]
Hello @AjayJagan how are you? What is the ticket/jira for this issue?

**Review by ederign — CHANGES_REQUESTED** [review]
Need more info about the proposed changes.

**Comment by porcelli** [comment]
As @ederign mentioned, a JIRA and more context are needed.

I'm also wondering how a login page here relates to Keycloak custom login page - I'm assuming that we will support KC only for security.

**Comment by AjayJagan** [comment]
> Hello @AjayJagan how are you? What is the ticket/jira for this issue?



> Need more info about the proposed changes.

Hello @ederign  . I am fine, How are you. I have been asked by Cristiano Nicolai to push the base code for Kogito UI(including dependencies,webpack configs, a basic login and landing page). That is the reason for creating the PR.

**Pull request closed without merging**